api/.env.example

```diff
@@ -94,5 +94,5 @@ GCNOTIFY_TEMPLATE_TEST_BULK_SMS="d662aac4-08bb-4917-860f-cc9999f4648d"
 # Feature flags
 FEATURE_ONGOING_RECRUITMENTS=true
 FEATURE_APPLICANT_DASHBOARD=true
-FEATURE_APPLICATION_REVAMP=true
+FEATURE_APPLICATION_REVAMP=false
```

Comment:
Should we make a note to change this back to true for dev once we fix the `e2e` stuff?

Reply:
Yeah, sure, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6458/commits/f59cb6d872c147e5af07fabe7169d5fa7a08502d
Will comment in the issue itself with a code snippet later.